pocs/tests/test_observatory.py

```diff
@@ -147,8 +166,7 @@ def test_default_config(observatory):
     """ Creates a default Observatory and tests some of the basic parameters """
 
     assert observatory.location is not None
-    assert observatory.location.get('elevation') - \
-        observatory.config['location']['elevation'] < 1. * u.meter
+    assert observatory.location.get('elevation') - observatory.config['location']['elevation'] < 1. * u.meter
```

Comment:
```suggestion
    assert observatory.location.get('elevation') == pytest.approx(observatory.config['location']['elevation'], rel=1 * u.meter)
```

I'm not actually sure if this will work with the units in place. Might need to just add `.value` to the end of each.

But the `pytest.approx` is the better way to test things like this.